perfmetrics/scripts/continuous_test/ml_tests/tf/resnet/build.sh

```diff
@@ -1,27 +1,27 @@
 #!/bin/bash
 
+# This will stop execution when any command will have non-zero status.
+set -e
+
 cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
 
-echo "Setting up the machine with Docker and Nvidia Driver"
+echo "Setting up the machine with Docker and Nvidia Driver..."
 chmod +x ml_tests/setup_host.sh
 source ml_tests/setup_host.sh
 
 cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/"
-
 mkdir container_artifacts && mkdir container_artifacts/logs && mkdir container_artifacts/output
 
-# Building the dlc based dockerfile
-sudo docker build . -f perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/Dockerfile -t tf-dlc-gcsfuse --build-arg DLC_IMAGE_NAME=tf-gpu.2-10
+echo "Building tf DLC docker image containing all tensorflow libraries..."
+sudo docker build . -f perfmetrics/scripts/ml_tests/tf/resnet/Dockerfile -t tf-dlc-gcsfuse --build-arg DLC_IMAGE_NAME=tf-gpu.2-10
 
-# Running the container image
+echo "Running the docker image build in the previous step..."
 sudo docker run --runtime=nvidia --name tf_model_container --privileged -d \
 -v ${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/container_artifacts/logs:/home/logs:rw,rshared \
 -v ${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/container_artifacts/output:/home/output:rw,rshared --shm-size=24g tf-dlc-gcsfuse:latest
 
-
-# Adding crontab entry
+echo "Setting up cron job to delete the gcsfuse_logs..."
```

Comment:
As we discussed, this log_deleter-script should be executed with sudo permission.

Reply:
Also, we should start keeping logs for 6 hrs, right now we keep the logs for 12 hrs.

Reply:
Please make the change for pytorch model also.